Translations/glossary.md

```diff
@@ -0,0 +1,66 @@
+# Translations glossary
+
+## Variables
+
+There are three types of variables in the text files: %s, %i, %d. All of these will add something within the single string, here are some examples. 
+
+`%s` will always be another word. Example: `%s of %s` refers to an created Item which includes affixes. `Tin Cap of the Fool` would be such an item.
+`%i` will always be a number. Example: Fire hit damage: %i-%i` will indicate how much fire damage a weapon will deal
+`%d` will always be a number. Example: `Player '%s' (level %d) just joined the game` - Player Arcas (level 15) just joined the game
```

Comment:
Maybe we should tell the developers to only use %i

Reply:
Sure. There are 10 occurances of "%d":
*note: multi.cpp has one string with three occurances.*

* [Source/DiabloUI/selconn.cpp:115](https://github.com/diasurgical/devilutionX/blob/ac6a398cb6c3d0ebdf72aa77a7139d7f16cd6305/Source/DiabloUI/selconn.cpp#L115)
* [Source/DiabloUI/selgame.cpp:415](https://github.com/diasurgical/devilutionX/blob/ac6a398cb6c3d0ebdf72aa77a7139d7f16cd6305/Source/DiabloUI/selgame.cpp#L415)
* [Source/appfat.cpp:117](https://github.com/diasurgical/devilutionX/blob/ac6a398cb6c3d0ebdf72aa77a7139d7f16cd6305/Source/appfat.cpp#L117)
* [Source/msg.cpp:2228](https://github.com/diasurgical/devilutionX/blob/ac6a398cb6c3d0ebdf72aa77a7139d7f16cd6305/Source/msg.cpp#L2228)
* [Source/multi.cpp:859](https://github.com/diasurgical/devilutionX/blob/ac6a398cb6c3d0ebdf72aa77a7139d7f16cd6305/Source/multi.cpp#L859)
* [Source/plrmsg.cpp:73](https://github.com/diasurgical/devilutionX/blob/ac6a398cb6c3d0ebdf72aa77a7139d7f16cd6305/Source/plrmsg.cpp#L73)
* [Source/qol/xpbar.cpp:124](https://github.com/diasurgical/devilutionX/blob/ac6a398cb6c3d0ebdf72aa77a7139d7f16cd6305/Source/xpbar.cpp#L124)
* [Source/qol/xpbar.cpp:150](https://github.com/diasurgical/devilutionX/blob/ac6a398cb6c3d0ebdf72aa77a7139d7f16cd6305/Source/xpbar.cpp#L150)

Reply:
%u is different as it can only be positive numbers